Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226993): Gdk-CRITICAL **: 08:46:51.343: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226928): Gdk-CRITICAL **: 08:46:51.507: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226697): Gdk-CRITICAL **: 08:46:52.477: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226877): Gdk-CRITICAL **: 08:46:53.160: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226703): Gdk-CRITICAL **: 08:46:53.192: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226690): Gdk-CRITICAL **: 08:46:53.204: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1227024): Gdk-CRITICAL **: 08:46:53.207: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226646): Gdk-CRITICAL **: 08:46:53.212: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226949): Gdk-CRITICAL **: 08:46:53.223: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226731): Gdk-CRITICAL **: 08:46:53.231: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226899): Gdk-CRITICAL **: 08:46:53.235: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226675): Gdk-CRITICAL **: 08:46:53.240: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226835): Gdk-CRITICAL **: 08:46:53.243: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226751): Gdk-CRITICAL **: 08:46:53.248: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226719): Gdk-CRITICAL **: 08:46:53.255: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226642): Gdk-CRITICAL **: 08:46:53.264: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226773): Gdk-CRITICAL **: 08:46:53.268: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226749): Gdk-CRITICAL **: 08:46:53.288: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226843): Gdk-CRITICAL **: 08:46:53.307: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226915): Gdk-CRITICAL **: 08:46:53.311: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226823): Gdk-CRITICAL **: 08:46:53.315: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226907): Gdk-CRITICAL **: 08:46:53.326: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1227002): Gdk-CRITICAL **: 08:46:53.331: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226923): Gdk-CRITICAL **: 08:46:53.338: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226827): Gdk-CRITICAL **: 08:46:53.344: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226821): Gdk-CRITICAL **: 08:46:53.355: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226755): Gdk-CRITICAL **: 08:46:53.363: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226857): Gdk-CRITICAL **: 08:46:53.372: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226654): Gdk-CRITICAL **: 08:46:53.375: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226729): Gdk-CRITICAL **: 08:46:53.380: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226996): Gdk-CRITICAL **: 08:46:53.384: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226867): Gdk-CRITICAL **: 08:46:53.390: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226989): Gdk-CRITICAL **: 08:46:53.394: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226975): Gdk-CRITICAL **: 08:46:53.406: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226757): Gdk-CRITICAL **: 08:46:53.411: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226959): Gdk-CRITICAL **: 08:46:53.417: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226706): Gdk-CRITICAL **: 08:46:53.427: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1227006): Gdk-CRITICAL **: 08:46:53.429: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226831): Gdk-CRITICAL **: 08:46:53.436: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226891): Gdk-CRITICAL **: 08:46:53.437: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226709): Gdk-CRITICAL **: 08:46:53.444: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226795): Gdk-CRITICAL **: 08:46:53.447: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226987): Gdk-CRITICAL **: 08:46:53.460: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226953): Gdk-CRITICAL **: 08:46:53.466: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226895): Gdk-CRITICAL **: 08:46:53.471: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226727): Gdk-CRITICAL **: 08:46:53.474: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226865): Gdk-CRITICAL **: 08:46:53.482: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226723): Gdk-CRITICAL **: 08:46:53.487: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226903): Gdk-CRITICAL **: 08:46:53.494: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226807): Gdk-CRITICAL **: 08:46:53.499: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226887): Gdk-CRITICAL **: 08:46:53.503: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226955): Gdk-CRITICAL **: 08:46:53.512: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226998): Gdk-CRITICAL **: 08:46:53.519: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226743): Gdk-CRITICAL **: 08:46:53.523: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226839): Gdk-CRITICAL **: 08:46:53.528: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226983): Gdk-CRITICAL **: 08:46:53.540: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:1226861): Gdk-CRITICAL **: 08:46:53.540: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226667): Gdk-CRITICAL **: 08:46:53.553: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:1226811): Gdk-CRITICAL **: 08:46:53.555: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226684): Gdk-CRITICAL **: 08:46:53.581: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:1226851): Gdk-CRITICAL **: 08:46:53.589: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226935): Gdk-CRITICAL **: 08:46:53.614: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:1226785): Gdk-CRITICAL **: 08:46:53.621: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226801): Gdk-CRITICAL **: 08:46:53.649: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1227010): Gdk-CRITICAL **: 08:46:53.657: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226913): Gdk-CRITICAL **: 08:46:53.669: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:1226945): Gdk-CRITICAL **: 08:46:53.672: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226714): Gdk-CRITICAL **: 08:46:53.677: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226911): Gdk-CRITICAL **: 08:46:53.689: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226937): Gdk-CRITICAL **: 08:46:53.709: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226769): Gdk-CRITICAL **: 08:46:53.713: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226875): Gdk-CRITICAL **: 08:46:53.722: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226746): Gdk-CRITICAL **: 08:46:53.742: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226797): Gdk-CRITICAL **: 08:46:53.745: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226849): Gdk-CRITICAL **: 08:46:53.773: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1227021): Gdk-CRITICAL **: 08:46:53.781: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226791): Gdk-CRITICAL **: 08:46:53.789: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226943): Gdk-CRITICAL **: 08:46:53.842: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226815): Gdk-CRITICAL **: 08:46:53.845: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226781): Gdk-CRITICAL **: 08:46:53.858: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226658): Gdk-CRITICAL **: 08:46:53.876: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226679): Gdk-CRITICAL **: 08:46:53.890: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226917): Gdk-CRITICAL **: 08:46:53.903: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:1226777): Gdk-CRITICAL **: 08:46:53.909: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1227008): Gdk-CRITICAL **: 08:46:53.917: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226692): Gdk-CRITICAL **: 08:46:53.926: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1227018): Gdk-CRITICAL **: 08:46:53.938: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226648): Gdk-CRITICAL **: 08:46:53.950: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226869): Gdk-CRITICAL **: 08:46:53.963: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226963): Gdk-CRITICAL **: 08:46:53.966: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226873): Gdk-CRITICAL **: 08:46:53.972: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1227022): Gdk-CRITICAL **: 08:46:53.974: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226739): Gdk-CRITICAL **: 08:46:54.000: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226734): Gdk-CRITICAL **: 08:46:54.011: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1227016): Gdk-CRITICAL **: 08:46:54.047: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1227015): Gdk-CRITICAL **: 08:46:54.103: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226971): Gdk-CRITICAL **: 08:46:54.151: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226991): Gdk-CRITICAL **: 08:46:54.212: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:1226763): Gdk-CRITICAL **: 08:46:54.505: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1226671): Gdk-CRITICAL **: 08:46:54.614: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
..........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
[1753 9401]
...
[1744 9390]
.
[1729 9412]
...
[1705 9453]
..
[1822 9338]
..
[1720 9442]
.
[1694 9422]
.
[1719 9412]
.
[1904 9244]
..
[1821 9344]
.
[1804 9380]
.
[1791 9355]
.
[1780 9336]
.
[1725 9443]
.
[1718 9389]
.
[1698 9426]
.
[1751 9343]
.
[1733 9414]
.
[1834 9273]
.
[1808 9332]
.
[1747 9416]
.
[1710 9421]
.
[1819 9326]
..
[1698 9430]
.
[1773 9361]
.
[1983 9147]
.
[1826 9306]
.
[1811 9379]
.
[1732 9379]
.
[1731 9361]
.
[1745 9318]
.
[1716 9408]
.
[1712 9433]
.
[1730 9430]
..
[1900 9200]
.
[1715 9403]
.
[1983 9092]
.
[1810 9295]
.
[1699 9426]
.
[1918 9185]
.
[1677 9413]
.
[1700 9415]
.
[1753 9349]
.
[1719 9374]
.
[1865 9254]
.
[2139 8875]
.
[1760 9389]
.
[1807 9310]
.
[1757 9396]
.
[1917 9180]
.
[2034 9052]
.
[2073 9016]
.
[1710 9389]
.
[2163 8906]
.
[1691 9431]
.
[1933 9124]
.
[2491 8459]
....
[2765 8139]
..
[2373 8601]
..
[2417 8531]
.
[2521 8405]
.
[2506 8423]
....
[3025 7856]
................
[2975 7889]
.
[3024 7815]
.....
[2925 8034]
........
[2881 8028]
...
[3251 7608]
........
[3486 7318]
........
[3395 7406]
...........
[3975 6777]
..
[3642 7083]
.
[3651 7076]
......
[4367 6291]
..................................................
[6080 4471]
...................................................................................................................................................................................................
Data Dump...
......
Data Dump...
...
Data Dump...
......
Data Dump...
...............
Data Dump...
.................
Data Dump...
.......................................................................................................................
[  54 5231]
................
[  50 5480]
.................
[  55 5186]
...............
[  55 4981]
......
[  74 4934]
..........
[  81 4885]
.....
[  93 4761]
.
[  71 4703]
........
[  87 4698]
...
[  68 4747]
...
[ 129 4434]
..............
[  81 4409]
.
[  80 4633]
..
[  78 4575]
.
[ 102 4338]
..
[  94 4471]
..........
[ 100 4508]
...................
[ 135 4215]
...
[ 118 4299]
......
[  97 4331]
.....
[ 134 4082]
........
[ 126 4169]
...
[ 152 3803]
......
[ 177 3958]
.....
[ 161 3783]
...
[ 136 3805]
.
[ 186 3930]
...
[ 175 3592]
...
[ 189 3866]
....
[ 170 3754]
....
[ 156 3968]
....
[ 194 3621]
..
[ 188 3767]
..
[ 194 3646]
.
[ 153 3667]
............
[ 238 3212]
.......
[ 222 3565]
.............
[ 244 3625]
.....
[ 254 3450]
..
[ 259 3116]
.............
[ 233 3596]
............
[ 291 3177]
...............
[ 317 2818]
.
[ 312 3177]
...........................
[ 374 2732]
..............
[ 445 2740]
.......................
[ 520 2234]
.....
[ 610 2035]
.................
[ 630 2143]
...
[ 658 2295]
.............
[ 584 2099]
............
[ 599 2271]
................
[ 718 2270]
..........
[ 705 2127]
......
[ 738 1890]
.............
[ 724 2322]
.
[1078 1316]
...
[ 872 1652]
..............
[ 930 1768]
.....
[ 784 2101]
.........
[ 32 446]
..................
[1029 1645]
.
[ 36 221]
...
[1056 1459]
.......
[1254 1304]
.
[1067 1580]
...
[ 33 462]
......
[1332 1289]
................
[ 35 412]
.................
[1329 1172]
..........
Data Dump...
.........
[2232  349]
......
[ 42 369]
....
[1438  868]
....
[1757  589]
...
Data Dump...

[ 49 390]
........
Data Dump...
....................
[ 39 277]
........
[ 41 310]
....
Data Dump...
...........
[ 41 324]
..............
[ 39 300]
.
[ 57 295]
..........................
Data Dump...
...
[ 48 296]
.......
[ 36 338]
.
Data Dump...
.
[ 42 276]
..............
[ 42 201]
..........
[ 51 164]
...
[ 43 293]
..........
Data Dump...
.....
[ 42 323]
.......
[ 57 263]
...
Data Dump...
........
Data Dump...
....
[ 31 307]
.............
Data Dump...
.........
[ 52 271]
..
Data Dump...
....
[ 59 213]
.......................
[ 51 244]
..
Data Dump...

Data Dump...
....
Data Dump...
......
[ 50 140]
...........
[ 65 201]
....
Data Dump...
.
[ 61 230]
...
[ 39 147]
.
Data Dump...

Data Dump...
....
[50 56]
.
[60 95]
...
[ 51 179]
........
Data Dump...
......
Data Dump...
......
[ 47 142]
...
[ 43 185]
.....
[ 53 143]
..
Data Dump...
...
[ 53 106]
..
[ 59 105]
..........
Data Dump...
...
[ 70 138]
.
[61 47]
......
Data Dump...
......
[ 66 189]
.......
[ 63 121]
.....
Data Dump...
....
[ 53 248]
.....................
Data Dump...
...
[52 61]
...
Data Dump...
......
Data Dump...
.
Data Dump...
.
Data Dump...

[52 95]
..
Data Dump...
.........
Data Dump...
.....
[ 57 133]
........
Data Dump...

Data Dump...
...............
Data Dump...

[54 30]
.
Data Dump...

Data Dump...
..........
Data Dump...
.
Data Dump...
....
[66 38]
..
Data Dump...
.....
[68 62]
...
Data Dump...

Data Dump...
...............
[71 18]
........
[79 57]
.
[68 27]
.
Data Dump...
..
[60 11]
....
Data Dump...
.....
[82 67]
.....
Data Dump...
.........
[71 15]
.
[75 78]
........
Data Dump...
..
[89 25]
...
[99  7]
.........
[89 71]
...
[77 21]
.
[82 12]
....
Data Dump...

[91 36]
....
Data Dump...
.
[92 47]
.............
[105  31]
...
[89 27]
..
[83 18]
...
[99 37]
.
[80 17]
..
Data Dump...
.
Data Dump...
....
Data Dump...
.
Data Dump...
.....
[103  20]
.
Data Dump...
...
[128  18]
..........
Data Dump...
..
[91  8]
.
Data Dump...
......
[106  10]
.......
Data Dump...
..
Data Dump...
.
Data Dump...
..
Data Dump...
.
Data Dump...
....
Data Dump...
...
Data Dump...
........
Data Dump...

Data Dump...

Data Dump...

Data Dump...
.
Data Dump...
....
Data Dump...

Data Dump...
...
Data Dump...
.
Data Dump...
Low 1 3
Done
slurmstepd-phcompute006: error: Detected 19 oom-kill event(s) in StepId=824568.batch. Some of your processes may have been killed by the cgroup out-of-memory handler.
